### PR TITLE
Use proper lldb path on Windows

### PR DESCRIPTION
**Pull request by matthewbastien**
Issue: #887 

On Windows, the LLDB executable is named `lldb.exe` and wasn't being located correctly.

### CODE REVIEW

**Review by award999 — APPROVED** [review]

**Comment by award999** [comment]
Is there existing tests we can add an extra case?

**Comment by matthewbastien** [comment]
> Is there existing tests we can add an extra case?

Not until we have a Windows testing platform on Jenkins.

File: src/toolchain/toolchain.ts [hunk 1]
```diff
@@ -346,6 +346,9 @@ export class SwiftToolchain {
      */
     public async getLLDB(): Promise<string> {
         let lldbPath = path.join(this.swiftFolderPath, "lldb");
```
Comment (compnerd) [inline src/toolchain/toolchain.ts:348]: How about sinking this into an `else` condition after the `win32` check? Alternatively, you could write this as a ternary:

```javascript
  let lldbPath = path.join(this.swiftFolderPath, process.platform === 'win32' ? 'lldb.exe' : 'lldb')
```

Comment (matthewbastien) [reply src/toolchain/toolchain.ts]: That works, too. Done!

**Review by adam-fowler — APPROVED** [review]

**Pull request merged**